dev/SearchBox.js

```diff
@@ -1,6 +1,9 @@
 import React, {Component, PropTypes as T} from 'react';
 import connect from '../src/connect.js';
 
-export default connect()(function({helper}) {
-  return <input onChange={(e) => helper.setQuery(e.target.value).search()} />;
+export default connect()(function({helper, search}) {
```

Comment:
Why not adding a new method on the helper like debouncedSearch() then? For now just in this project.